WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/taxes/rates/TaxRateSelectorScreen.kt

```diff
@@ -117,24 +179,36 @@ private fun TaxRates(
         item {
             Header(onInfoIconClicked)
         }
-        item {
-            Text(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(
-                        horizontal = dimensionResource(id = R.dimen.major_100),
-                        vertical = dimensionResource(
-                            id = R.dimen.minor_100
-                        )
-                    ),
-                fontSize = 13.sp,
-                color = colorResource(id = R.color.woo_gray_40),
-                text = stringResource(R.string.tax_rate_selector_list_header)
-            )
-            Divider()
-        }
-        itemsIndexed(state.taxRates) { _, taxRate ->
-            TaxRateRow(taxRate, onTaxRateClick)
+        when {
+            state.taxRates.isEmpty() -> {
```

Comment:
```suggestion
            state.taxRates.isEmpty() && !state.isLoading -> {
```

This would be a nice improvement, to avoid a glitch before tax rates are loaded. Currently, the empty state is shown briefly during the initial fetching of the rates and then replaced by the rates list. 

We could also move this logic to the ViewState class:
```kotlin
data class ViewState(
  ...
  val isEmpty: Boolean = taxRates.isEmpty() && !isLoading
)
```

Reply:
Done here https://github.com/woocommerce/woocommerce-android/pull/9759/commits/d0c487f9318740d5550d9916b982d5b1763fe8d4